cmd/commands/cli/command.go

```diff
@@ -745,3 +746,12 @@ func parseStartFlags(serviceType string, args ...string) (services.StartOptions,
 
 	return services.GetStartOptions(serviceType)
 }
+
+func validServiceType(s string) bool {
```

Comment:
I would rather include this validation in `services` package - it's more relevant there than in CLI package.